news/477.feature

```diff
@@ -0,0 +1 @@
+Perf improvement: eliminated unnecessary deepcopy calls during construction
```

Comment:
Not sure if it matters, but this is also a potentially breaking change (if I understand correctly), because of something like:
```python
bar = OmegaConf.create({"zonk": 0})
cfg.foo = bar
bar.zonk = 1
assert cfg.foo.zonk == 0  # would fail now?
```
I didn't test it though so maybe that's not the case?

Reply:
Ok I'll have to look at the updated code more closely, but as long as you're aware of it that sounds fine for now => resolving